tests/lib/sms/src/sms_test.c

```diff
@@ -419,6 +419,12 @@ void test_send_len3_number10plus(void)
 {
 	sms_reg_helper();
 
+	__mock_nrf_modem_at_scanf_ExpectAndReturn(
+		"AT+CRSM=176,28589,0,0,0", "+CRSM: %d, %d, \"%511[^\"]\"", 3);
+	__mock_nrf_modem_at_scanf_ReturnVarg_int(144);
+	__mock_nrf_modem_at_scanf_ReturnVarg_int(0);
+	__mock_nrf_modem_at_scanf_ReturnVarg_string("00FFFF02");
+
```

Comment:
This same piece of code is present in ~15 test cases. Just wondering if it could be in a function instead. Unless you really want to have all steps visible in each test case.

Reply:
So far I've preferred to see the steps in one place within a test function.
And I've done changes in other Unity test sets where, e.g., a change from `nrf_modem_at_scanf` to `nrf_modem_at_cmd`, or vice versa, has occurred and I had to change all those occurrences. Search and replace is a nice friend 😄

Reply:
Ok, fair enough. You will probably maintain the tests anyway. 😄